templates/rapidsai/partials/cleanup-chmod.dockerfile.j2

```diff
@@ -1,5 +1,6 @@
 {# This partial file is used for cleanup and ACL changes at the end of each image #}
 
 {# Clean-up conda and update ACLs for all users #}
-RUN conda clean -afy \
+RUN chmod -R ugo+w /opt/conda ${RAPIDS_DIR} ${BLAZING_DIR} \
+  && conda clean -tipsy \
```

Comment:
You removed the `-s` option everywhere except there. Is it expected?

Reply:
Good catch, the `-s` option was deprecated so I'll update this one as well